tox.ini

```diff
@@ -0,0 +1,77 @@
+[tox]
+envlist = py37
+isolated_build = true
+skipsdist=True
```

Comment:
Why are these two options necessary?

Reply:
By default, Tox depends on either setup.py and pyproject.toml for installation and setup purposes. Since we want our tox env to be independent of such dependencies, theses options help to disable that

Reply:
OK thanks.